### ISSUE TITLE
i got a file called 'api.pb.micro.go',but not 'api.micro.go'.

### ISSUE THREAD

**Issue by jasionLaw** [micro/protoc-gen-micro#57]
Follow the documentation，but i can not get the file。

> `protoc --plugin=protoc-gen-go=/Users/mike/golang/bin/protoc-gen-go --plugin=protoc-gen-micro=/Users/mike/golang/bin/protoc-gen-micro --proto_path=/Users/mike/golang/src/github.com/micro/go-micro/api/ --go_out=. --micro_out=. proto/api.proto`

then i will get `api.pb.micro.go`

> // Code generated by protoc-gen-micro. DO NOT EDIT.
>
>// source: proto/api.proto
>
>package go_api
>
>import (
>	fmt "fmt"
>	proto "github.com/golang/protobuf/proto"
>	math "math"
>)
>
>// Reference imports to suppress errors if they are not otherwise used.
>var _ = proto.Marshal
>var _ = fmt.Errorf
>var _ = math.Inf
>
>// This is a compile-time assertion to ensure that this generated file
>// is compatible with the proto package it is being compiled against.
>// A compilation error at this line likely means your copy of the
>// proto package needs to be updated.
>const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package

**Comment by vtolstov** [comment]
yes, this is newest change to have generated files names like other tools do. xx.pb.plugin.go

**Comment by jasionLaw** [comment]
> yes, this is newest change to have generated files names like other tools do. xx.pb.plugin.go

but, it has not func like Register****,***Handler.I am so confused about this. Maybe give  a doc?

**Comment by vtolstov** [comment]
this is strange, we don't change codegeneration
